### PR TITLE
[BLOG] Add uarray-attempting-to-move-the-ecosystem-forward

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-uarray-attempting-to-move-the-ecosyst-fec914-quansight.vercel.app/blog/uarray-attempting-to-move-the-ecosystem-forward).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: 'uarray: Attempting to move the ecosystem forward'
+published: November 10, 2019
+author: hameer-abbasi
+description: 'There comes a time in every project where most technological hurdles have been surpassed, and its adoption is a social problem. I believe uarray and unumpy had reached such a state, a month ago. I'd like to present some improvements the Quansight team has made to uarray over the course of the past several months.'
+category: [PyData Ecosystem]
```
Comment (MarsBarLee) [inline apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md:6]:
```suggestion
category: [PyData ecosystem]
```

File: apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: 'uarray: Attempting to move the ecosystem forward'
+published: November 10, 2019
+author: hameer-abbasi
+description: 'There comes a time in every project where most technological hurdles have been surpassed, and its adoption is a social problem. I believe uarray and unumpy had reached such a state, a month ago. I'd like to present some improvements the Quansight team has made to uarray over the course of the past several months.'
```
Comment (MarsBarLee) [inline apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md:5]:
```suggestion
description: "There comes a time in every project where most technological hurdles have been surpassed, and its adoption is a social problem. I believe uarray and unumpy had reached such a state, a month ago. I'd like to present some improvements the Quansight team has made to uarray over the course of the past several months."
```

File: apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+title: 'uarray: Attempting to move the ecosystem forward'
+published: November 10, 2019
+author: hameer-abbasi
+description: "There comes a time in every project where most technological hurdles have been surpassed, and its adoption is a social problem. I believe uarray and unumpy had reached such a state, a month ago. I'd like to present some improvements the Quansight team has made to uarray over the course of the past several months."
+category: [PyData ecosystem]
+featuredImage:
+  src: /posts/uarray-attempting-to-move-the-ecosystem-forward/blog_feature_var1.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/uarray-attempting-to-move-the-ecosystem-forward/blog_feature_var2.svg
```
Comment (MarsBarLee) [inline apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md:11]:
```suggestion
  imageSrc: /posts/uarray-attempting-to-move-the-ecosystem-forward/blog_feature_var2.svg
```
```suggestion
  imageSrc: /posts/uarray-attempting-to-move-the-ecosystem-forward/blog_hero_var2.svg
```

File: apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+I must confess I felt discouraged at some points in the `uarray` journey: However, realising my flaws will make me perform better in the future.
+
+Although my main focus at this point in my career isn't `uarray` (not that I don't want it to be, only that social problems don't take very much of your time), it isn't a small library at the back of my head by any means.
+
+With that out of the way, I'd like to present some improvements the Quansight team has made to `uarray` over the course of the past several months:
+
+## Progress on Integration
+### `uarray` ❤️ `scipy.fft`
+`uarray` is now the [official override mechanism](http://scipy.github.io/devdocs/fft.html#backend-control) for `scipy.fft`. This is a *huge* win.
```
Comment (noatamir) [inline apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md:29]:
```suggestion
`uarray` is now the [official override mechanism](https://scipy.github.io/devdocs/reference/fft.html#backend-control) for `scipy.fft`. This is a *huge* win.
```

Comment (noatamir) [reply apps/labs/posts/uarray-attempting-to-move-the-ecosystem-forward.md:29]: fix link

**Review by noatamir — APPROVED** [review]
LGTM

**Pull request merged**